src/clients/AcrossConfigStoreClient.ts

```diff
@@ -241,6 +245,9 @@ export class AcrossConfigStoreClient {
       paginatedEventQuery(this.configStore, this.configStore.filters.UpdatedGlobalConfig(), searchConfig),
     ]);
 
+    // Events *should* normally be received in ascending order, but explicitly enforce the ordering.
+    [updatedTokenConfigEvents, updatedGlobalConfigEvents].forEach((events) => sortEventsAscendingInPlace(events));
+
```

Comment:
QQ: Will sorting this in place cause any external issues regarding how we treat these variables?

Reply:
The objects have a relatively short lifespan - they're populated in `_update()`, and are subsequently iterated over in order to transform them into Across-specific objects. There is an implicit assumption in the code that the events are received in ascending order, so I don't see any possible negative side-effects from enforcing the ordering here.